hera_mc/mc_session.py

```diff
@@ -825,6 +825,72 @@ def get_weather_data(self, starttime, stoptime=None, variable=None):
                                  stoptime=stoptime, filter_column='variable',
                                  filter_value=variable)
 
+    def add_roach_temperature(self, time, roach, ambient_temp, inlet_temp,
+                              outlet_temp, fpga_temp, ppc_temp):
+        """
+        Add new roach temperature to the M&C database.
+
+        Parameters:
+        ------------
+        time: astropy time object
+            astropy time object based on a timestamp from the katportal sensor.
+        roach: int
+            roach number
```

Comment:
Perhaps these should be strings? One could imagine a future where there's not a clear and easy numbering of the ROACHes.

Reply:
sure, I'll change it.